File: backend/src/Application/DTO/Movimiento/EstadoCuentaResult.cs
namespace Application.DTO.Movimiento;

public class EstadoCuentaResult
{
  public int ClienteId { get; set; }
  public string Cliente { get; set; } = default!;
  public DateTime Desde { get; set; }
  public DateTime Hasta { get; set; }
  public List<EstadoCuentaCuentaDto> Cuentas { get; set; } = new();
  public string? PdfBase64 { get; set; }
}

File: frontend/banking-app/src/app/shared/models/estadoCuentaResult.ts
// @ts-ignore
import {CuentaReporte} from '@app/shared/models/cuentaReporte';


export interface EstadoCuentaResult {
  clienteId: number;
  cliente: string;
  desde: string;
  hasta: string;
  cuentas: CuentaReporte[];
  pdfBase64?: string | null;
}
